### DISCUSSION TITLE
Welcome screen with basic keybinding and last opened files.

### DISCUSSION THREAD

**Discussion by debajyoti1990** [helix-editor/helix#6366] [Ideas]
Has this ever come into consideration ? Basic keybinding would help new terminal editor users, besides asthetics. Not every person comes to Helix from (n) vim/ kakoune.

**Comment by askreet** [comment]
I like the idea of a landing screen for bare `hx` invocations, similar to what LunarVim and other configurations do. One option could be a link to the tutor document ("New? Press `?` to open Helix Tutor.")

**Comment by gabydd** [comment]
You can see some discussion here as well: https://github.com/helix-editor/helix/issues/2339

**Reply by debajyoti1990** [reply]
Thanks. I searched for the same in issues and PRs, couldn't find any, hence started a discussion thread.
